cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -385,15 +388,29 @@ func (s *CMServer) Run(_ []string) error {
 		}
 	}
 
+	volumePlugins := ProbeRecyclableVolumePlugins(s.VolumeConfigFlags)
+	provisioner, provisionerExpected, err := NewVolumeProvisioner(volumePlugins, cloud, s.VolumeConfigFlags)
+	if provisionerExpected && err != nil {
+		glog.Fatalf(fmt.Sprint("%v", err))
```

Comment:
I'd recommend either logging some additional context for the error, or just doing `glog.Fatal(err)`